template-README.md

```diff
@@ -256,7 +279,7 @@ The code is licensed under a MIT/BSD/GPL [choose one!] license. See [LICENSE.txt
 - Download the data files referenced above. Each should be stored in the prepared subdirectories of `data/`, in the format that you download them in. Do not unzip. Scripts are provided in each directory to download the public-use files. Confidential data files requested as part of your FSRDC project will appear in the `/data` folder. No further action is needed on the replicator's part.
```

Comment:
are those instructions specific to your AEA setup or is this a generic example? what's FSRDC? If generic, isn't that a bit misleading? _do i have to create a FSRCD folder in my replication package_ could be one question.

Reply:
It does say "Examples follow" right above it...